cypress/support/fragments/inventory/inventorySearch.js

```diff
@@ -0,0 +1,23 @@
+import { MultiColumnList, Checkbox, MultiColumnListCell, MultiSelect } from '../../../../interactors';
+
+export default class InventorySearch {
+  static #effectiveLocationInput = MultiSelect({ 'id': 'multiselect-6' });
+  static effectiveLocation = {
+    // add new effective location inputs as needed
+    'mainLibrary': 'Main Library'
+  }
+
+  static #firstResultCheckbox = MultiColumnListCell({ 'row': 0, 'columnIndex': 0 }).find(Checkbox());
+
+  static getFirstResultCheckbox() {
+    return this.#firstResultCheckbox;
+  }
+
+  static getSearchResults() {
+    return MultiColumnList();
+  }
+
+  static byEffectiveLocation(values = [this.effectiveLocation.mainLibrary]) {
+    return InventorySearch.#effectiveLocationInput.select(values);
+  }
+}
```

Comment:
Since all of these are just static constants, why use a class here at all? Why not just

```js
export default {
    effectiveLocationInput: MultiSelect({ 'id': 'multiselect-6' }),
    effectiveLocation:  {
      'mainLibrary': 'Main Library'
    },
    firstResultCheckbox: MultiColumnListCell({ 'row': 0, 'columnIndex': 0 }).find(Checkbox()),
 
    getFirstResultCheckbox: get() {
      return this.firstResultCheckbox;
    },
 
    getSearchResults() {
      return MultiColumnList();
    }
 
    byEffectiveLocation(values = [this.effectiveLocation.mainLibrary]) {
      return InventorySearch.#effectiveLocationInput.select(values);
    }
}
```

But honestly, that begs the question, why not just make these individual exports? E.g.

```js
export const effectiveLocationInput = MultiSelect({ 'id': 'multiselect-6' });
```

Reply:
Then, you can import a single symbol if that's all you're going to use.

```js
import { effectiveLocationInput } from '.../fragments/inventory/inventorySearch`;
```

or if you want to slurp the whole module, and use it as before:

```js
import * as InventorySearch from '../fragments/inventory/inventorySearch';

InventorySearch.effectiveLocationInput
```

Reply:
in case of class usage code will have more special name like InventorySearch.byEffectiveLocation() instead of just byEffectiveLocation(). Class wraps all related with special page object into one place

Reply:
> code will have more special name like InventorySearch.byEffectiveLocation() instead of just byEffectiveLocation().

As my code example shows though, you don't need classes to do this. The class syntax is just noise. Also, if you export standalone functions, you can still use the `InventorySearch.byEffectiveLocation()`. It's more flexible because you have the option to do both.